.gitignore

```diff
@@ -10,68 +9,63 @@ cabal-dev
 *.aux
 *.hp
 *.tix
-.virtualenv
-.hsenv
-.hpc
-.Werror
-*.svg
-.stack-work/
+
+# Cabal
+cabal-dev
 .cabal-sandbox/
 cabal.sandbox.config
 cabal.config
-GTAGS
-GRTAGS
-GPATH
-TAGS
-.DS_Store
-*~
-*#
-.dir-locals.el
+
+# Stack
+.stack-work/
+
+# From daedalus-bridge
+node_modules/*
+
+# Config files
+.ghci
+.hindent.yaml
+.stylish-haskell.yaml
+
+# Our custom build script (util-scripts/build.sh)
+b
+.no-nix
+.ram
+.Werror
+
+# Various DBs
 node-db*
 pos-db*
 cardano-db*
-*.html
-pos-keys
-nohup.out
-logs
-stats-node-*
-testRun
-*.csv
-my-log-template.yaml
-tags
-run
-*.swp
-.ghci
-*.key
-!secrets/*.key
 wallet-db/
-zhogovo/
-smartgen-secret.sk
-rocks-smartwallet/
-cardano-keys/
-bla/
-testnetStakes/
-*.key.lock
-node_modules/*
 db-abc/
```

Comment:
It's obsolete I suppose. Grep finds `abc` in scripts only in `scripts/avvm-files/utxo-dump-last-new.json` and `scripts/clean/db.sh`.